polaris.shopify.com/src/utils/search.ts

```diff
@@ -1,11 +1,14 @@
 import { SearchResult } from "../types";
 import { createVar, tokens } from "@shopify/polaris-tokens";
-import components from "../data/components.json";
-import icons from "../data/icons.json";
-import guidelines from "../data/guidelines.json";
 import Fuse from "fuse.js";
 import { slugify, stripMarkdownLinks } from "./various";
 
+import components from "../data/components.json";
+import guidelines from "../data/guidelines.json";
+import {Icons} from '../types';
+
+const icons: Icons = require('@shopify/polaris-icons/metadata');
```

Comment:
The module mixing 🙈 this pattern of `require` & `import` together scares me a bit 😬